var/spack/repos/builtin/packages/c-lime/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
```

Comment:
Did you delete the first line of the license?

Reply:
ops, yes, did it by accident. fixed in d65a7dc